contrib/pg_tde/src/catalog/tde_keyring.c

```diff
@@ -184,15 +191,28 @@ cleanup_key_provider_info(Oid databaseId)
 	PathNameDeleteTemporaryFile(kp_info_path, false);
 }
 
+static Datum
+			pg_tde_change_key_provider_c(PG_FUNCTION_ARGS, Oid dbOid, int shift);
+
 Datum
 pg_tde_change_key_provider_internal(PG_FUNCTION_ARGS)
 {
-	char	   *provider_type = text_to_cstring(PG_GETARG_TEXT_PP(0));
-	char	   *provider_name = text_to_cstring(PG_GETARG_TEXT_PP(1));
-	char	   *options = text_to_cstring(PG_GETARG_TEXT_PP(2));
-	bool		is_global = PG_GETARG_BOOL(3);
+	return pg_tde_change_key_provider_c(fcinfo, MyDatabaseId, 0);
```

Comment:
Why not rename `pg_tde_change_key_provider_internal` to `pg_tde_change_key_provider` and  `pg_tde_change_key_provider_c` to `pg_tde_change_key_provider_internal`?

Reply:
These comments apply to all 5 functions.

Reply:
Done.